purchase_reception_status/views/purchase_order.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+  Copyright 2020 Akretion France (http://www.akretion.com/)
+  @author: Alexis de Lattre <alexis.delattre@akretion.com>
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+<odoo>
+    <record id="purchase_order_form" model="ir.ui.view">
+        <field name="name">received_status.purchase.order.form</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_form" />
+        <field name="arch" type="xml">
+            <field name="invoice_status" position="before">
+                <field name="reception_status" states="purchase,done" />
+                <field
+                    name="force_received"
+                    groups="purchase.group_purchase_manager"
+                    states="purchase,done"
+                />
+            </field>
+        </field>
+    </record>
+    <record id="purchase_order_tree" model="ir.ui.view">
+        <field name="name">received_status.purchase.order.tree</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_tree" />
+        <field name="arch" type="xml">
+            <field name="invoice_status" position="before">
+                <field name="reception_status" />
```

Comment:
I think this should be aligned with invoice_status layout and new badge layout option:
                      ```
 widget="badge"
                       decoration-info="reception_status == 'partial'" decoration-success="reception_status == 'received'"/>
```

Reply:
@saxomanu can you apply this? Thanks!